x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/settings/default_settings_flyout.tsx

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { docLinks } from '../../../shared/doc_links';
+
+import { SettingsLogic } from './settings_logic';
+import { SettingsPanel } from './settings_panel';
+
+export interface DefaultSettingsFlyoutProps {
+  closeFlyout: () => void;
+}
+
+const Callout = (
+  <EuiCallOut
+    title={i18n.translate('xpack.enterpriseSearch.defaultSettingsFlyout.callout.title', {
+      defaultMessage: 'Individual settings management',
+    })}
+  >
+    {i18n.translate('xpack.enterpriseSearch.defaultSettingsFlyout.callout.body', {
+      defaultMessage:
+        'You can also enable or disable this feature for a specific index on the index’s configuration page.',
+    })}
+  </EuiCallOut>
+);
+export const DefaultSettingsFlyout: React.FC<DefaultSettingsFlyoutProps> = ({ closeFlyout }) => {
+  const { makeRequest, setPipeline } = useActions(SettingsLogic);
+  const { defaultPipeline, hasNoChanges, isLoading, pipelineState } = useValues(SettingsLogic);
+  const {
+    extract_binary_content: extractBinaryContent,
+    reduce_whitespace: reduceWhitespace,
+    run_ml_inference: runMLInference,
+  } = pipelineState;
+  return (
+    <EuiFlyout onClose={closeFlyout} size="s" paddingSize="l">
```

Comment:
Looking at the design should the size of this be `size="m"` instead?

Reply:
Size M is too big compared to designs. 🤷‍♂️